code/modules/organs/organ.dm

```diff
@@ -287,13 +289,16 @@
 
 /obj/item/organ/proc/rejuvenate(var/ignore_prosthetic_prefs)
 	damage = 0
-	status = initial(status)
 	if(ignore_prosthetic_prefs && ishuman(owner) && owner.client && owner.client.prefs && owner.client.prefs.real_name == owner.real_name)
 		for(var/decl/aspect/aspect as anything in owner.personal_aspects)
 			if(aspect.applies_to_organ(organ_tag))
 				aspect.apply(owner)
-	if(species)
-		species.post_organ_rejuvenate(src, owner)
+	reset_status()
+
+/obj/item/organ/proc/reset_status()
+	status = initial(status)
```

Comment:
This isn't very reliable. Because initial is the state as defined in the definition. It doesn't take into account things done in init.

Also, it a bit silly to do all that extra processing just because you don't wanna move mostly immutable flags to their own var, and literally not have to do any processing at all besides when reseting the mob completely. Like prosthetic and crystal definitely shouldn't be stored along with mutable flags like dislocated, artery cut, cut away, etc..

Also shouldn't you reset the status var before applying aspects? Because you're just gonna clear whatever you've applied to the status?

Reply:
I am replicating the existing behavior, not adding new behavior. If you look at the diff you'll see that previously it set it to 0, which would have cleared any init behavior as well.
Good point about aspects, I actually moved it up higher.

I sincerely have no clue what you mean by `all that extra processing`. This PR has absolutely nothing to do with splitting properties and status into two args, that was your PR and I don't object to it.

Reply:
Well, its just you've been working on exactly the same part as I've been and fixing the same issue. So I've assumed you probably don't want the changes you've excluded from my stuff? 

> I also don't like this, though maybe for a different reason. I don't think you need two vars for the flags, but you probably should be doing something like `status &= (ROBOTIC | CRYSTAL)` here. If you have a prosthetic organ, won't this de-robotize it otherwise?

I think its important to separate mutable from immutable stuff? Just because its easy to accidentally decide to clear that var named "status" assuming it actually only contains status effects, and not properties? But whatever.

Reply:
The only changes I don't want from your PR are the ones I mentioned as blocking on Discord, which I was making this alternative PR for, specifically the subtyping of all organs and the removal of the post-species modifier proc. Everything else in your PR is fine, I don't disagree conceptually with any of your thoughts on the flag splitting/clustering.